packages/fxa-auth-server/lib/metrics/glean/index.ts

```diff
@@ -228,13 +228,13 @@ const createEventFn =
         case 'access_token_checked':
           gleanServerEventLogger.recordAccessTokenChecked(commonMetrics);
           break;
-        case 'google_login_complete':
+        case 'third_party_auth_google_login_complete':
           gleanServerEventLogger.recordThirdPartyAuthGoogleLoginComplete({
```

Comment:
Aside from the rename, for some reason this method doesn't seem to be executing properly in the deployed code since we [do not receive](https://mozilla.cloud.looker.com/dashboards/1452?Event+Name=%22third_party_auth.google_login_complete%22&App+Name=Mozilla+Accounts+Backend&Window+Start+Time=28+days&Channel=) any of these events.
Surprisingly, a call below on line 250 (`await gleanEventLogger.record({...`) does produce events.